go.mod

```diff
@@ -1,12 +1,87 @@
 module terraform-provider-maas
 
-go 1.16
+go 1.19
```

Comment:
is 1.19 actually required or would 1.18 suffice? The reason I ask is we tend to use the default deb installation of Go to build, and that happens to be 1.18 at the moment. We can work around that, but matching the versions would be easier.